packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -65,7 +70,7 @@ function getNormalizedContents(
   replacements: generatorCommon.Replacements,
 ) {
   // Template files are CRLF, JS-generated replacements are LF, normalize replacements to CRLF
-  for (var key in replacements) {
+  for (const key of Object.keys(replacements)) {
```

Comment:
eslint will warn about “for in” without hasOwnProperty checks since we almost never want the behavior of prototype chain traversal.